bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -202,16 +201,21 @@ public String getValue(final Row row) {
 
             @Override public Option<GuideGroup> group() { return group; }
             @Override public List<Row> children()       { return children; }
-            @Override public Icon getIcon()             { return groupIcon; }
         }
 
+        // Collection of rows, which may include "subrows".
         private final ImList<Row> rows;
+
+        // Total number of rows, including subrows. Precalculated as swing uses this value frequently.
+        private final int numRows;
```

Comment:
It is a bit confusing (just as it was before this PR too) that "row" sometimes means a normal table row and sometimes means a group of rows.  I think it's just a terminology thing.  I shouldn't have called the `Target \/ Group` concept a "Row" all those years ago.  Not sure what else to suggest though.